admin-tools/lambda/src/main/scala/com/gu/mediaservice/BatchIndexHandlerAwsFunctions.scala

```diff
@@ -0,0 +1,88 @@
+package com.gu.mediaservice
+
+import java.io.ByteArrayInputStream
+import java.nio.ByteBuffer
+import java.util.UUID
+
+import com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient
+import com.amazonaws.services.dynamodbv2.document.{Table, DynamoDB => AwsDynamoDB}
+import com.amazonaws.services.kinesis.model.PutRecordRequest
+import com.amazonaws.services.kinesis.{AmazonKinesis, AmazonKinesisClientBuilder}
+import com.amazonaws.services.s3.AmazonS3ClientBuilder
+import com.amazonaws.services.s3.model.ObjectMetadata
+import com.gu.mediaservice.lib.json.JsonByteArrayUtil
+import play.api.libs.json.JsValue
+
+class BatchIndexHandlerAwsFunctions(cfg: BatchIndexHandlerConfig) {
```

Comment:
Could we reuse the AWS helpers in [`common-lib`](https://github.com/guardian/grid/tree/master/common-lib/src/main/scala/com/gu/mediaservice/lib/aws)?

Reply:
We also have this [message publisher](https://github.com/guardian/grid/blob/f8b6c7d63de84fa9813db537d034322aaa2751de/common-lib/src/main/scala/com/gu/mediaservice/lib/aws/ThrallMessageSender.scala#L16).

Reply:
i was looking into that but they have a bit different use case, the use well with EC2
were you have creds chain profile -> instance profile for lambda the best solution is just standard client without any creds as it will use execution role, but i can double check